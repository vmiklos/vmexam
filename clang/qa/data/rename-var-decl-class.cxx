class C
{
public:
    static const int aS[];
};

int main()
{
    C* pC = 0;
    return 0;
}
