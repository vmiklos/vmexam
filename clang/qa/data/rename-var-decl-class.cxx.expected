class D
{
public:
    static const int aS[];
};

int main()
{
    D* pC = 0;
    return 0;
}
